content/hi/docs/setup/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+
+
+
+
+title: शुरू करना
+main_menu: true
+weight: 20
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#सीखने का माहौल"
+    title: सीखने का माहौल
+  - anchor: "#उत्पादन वातावरण"
+    title: उत्पादन वातावरण
+---
+
+<!-- overview -->
+
+यह भाग कुबेरनेट्स को स्थापित करने और चलाने के विभिन्न तरीकों को सूचीबद्ध करता है।
+जब आप कुबेरनेट्स स्थापित करते हैं, तो इसके आधार पर एक इंस्टॉलेशन प्रकार चुनें: रखरखाव में आसानी, सुरक्षा,
+क्लस्टर के संचालन और प्रबंधन के लिए आवश्यक नियंत्रण, उपलब्ध संसाधन और विशेषज्ञता।
+
+कुबेरनेट्स क्लस्टर को परिनियोजित करने के लिए आप [कुबेरनेट्स डाउनलोड](/releases/download/) कर सकते हैं
+स्थानीय मशीन पर, क्लाउड में, या अपने स्वयं के डेटासेंटर के लिए।
+
+यदि आप कुबेरनेट्स क्लस्टर को स्वयं संभालना नहीं चाहते हैं, तो आप एक संचालन सेवा चुन सकते हैं, जिसमें शामिल हैं [प्रमाणित प्लेटफॉर्म](/docs/setup/production-environment/turnkey-solutions/)।
+क्लाउड की विस्तृत श्रृंखला में अन्य मानकीकृत और कस्टम समाधान भी हैं और
+नंगे धातु वातावरण।
+
+<!-- body -->
+
+## सीखने का माहौल
+
+यदि आप कुबेरनेट्स सीख रहे हैं, तो कुबेरनेट्स समुदाय द्वारा समर्थित टूल का उपयोग करें,
+या स्थानीय मशीन पर कुबेरनेट्स क्लस्टर स्थापित करने के लिए पारिस्थितिकी तंत्र में उपकरण।
+[इंस्टॉल टूल](/docs/tasks/tools/) देखें।
+
+## उत्पादन वातावरण
+
+समाधान का मूल्यांकन करते समय [उत्पादन परिवेश](/docs/setup/production-environment/), विचार करें कि के किन पहलुओं पर विचार करें
+कुबेरनेट्स क्लस्टर (या _abstractions_) का संचालन करना जिसे आप स्वयं प्रबंधित करना चाहते हैं और जिसे आप
+एक प्रदाता को सौंपना पसंद करते हैं।
+
+उस क्लस्टर के लिए जिसे आप स्वयं प्रबंधित कर रहे हैं, आधिकारिक रूप से समर्थित टूल
+कुबेरनेट्स की तैनाती के लिए है [कुबेदम](/docs/setup/production-environment/tools/kubeadm/)
+
+## {{% heading "whatsnext" %}}
+
+- [कुबेरनेट्स डाउनलोड करें](/releases/download/)
+- डाउनलोड और [उपकरण स्थापित करें](/docs/tasks/tools/) `kubectl` सहित।
+- अपने नए क्लस्टर के लिए [कंटेनर रनटाइम](/docs/setup/production-environment/container-runtimes/) चुनें
+- क्लस्टर सेटअप के लिए [सर्वोत्तम प्रक्रियाओं](/docs/setup/best-practices/) के बारे में जानें
+
+
+कुबेरनेट्स को इसके {{< glossary_tooltip term_id="control-plane" text="control plane" >}} के लिए Linux पर चलाने के लिए डिज़ाइन किया गया है। अपने क्लस्टर के भीतर आप लिनक्स या अन्य ऑपरेटिंग सिस्टम पर एप्लिकेशन चला सकते हैं, जिसमें शामिल हैं विंडोज़।
```

Comment:
nit: I would transcribe or translate the text in `text="control plane"`

Reply:
oops, i have lost this pull request branch from my computer so should i leave it as it is or i should create new PR and then will close this PR.

Reply:
You have it on GitHub. Right?

Reply:
ok @vedant-kakde